packages/types/src/KeyValue.spec.js

```diff
@@ -5,11 +5,11 @@
 import KeyValue, { KeyValueOption } from './KeyValue';
 
 describe('KeyValue', () => {
-  it('decodes KeyValue from u8a', () => {
+  it.skip('decodes KeyValue from u8a', () => {
```

Comment:
If StorageKey is not length-prefixed, there's not way we can decode from u8a. Fortunately we don't need the decoding for KeyValue.